hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/EmptyInventory.java

```diff
@@ -1170,4 +1194,99 @@ public Relationship associationWith(Path path) throws RelationNotFoundException
             return new ResourcesSingle();
         }
     }
+
+    public static class DatasRead implements Datas.Read {
+
+        @Override
+        public Datas.Multiple getAll(Filter[][] filters) {
+            return new DatasMultiple();
+        }
+
+        @Override
+        public Datas.Single get(Void ignored) throws EntityNotFoundException {
+            return new DatasSingle();
+        }
+
+        @Override
+        public Resources.Single resource() {
+            return new ResourcesSingle();
+        }
+    }
+
+    public static class DatasReadWrite implements Datas.ReadWrite {
+
+        @Override
+        public Resources.Single resource() {
+            return new ResourcesSingle();
+        }
+
+        @Override
+        public Datas.Multiple getAll(Filter[][] filters) {
+            return new DatasMultiple();
+        }
+
+        @Override
+        public Datas.Single get(Void ignored) throws EntityNotFoundException {
```

Comment:
pitta, Java hasn't support for the default parameters... the `.get(null)` in the API looks ugly